x-pack/plugins/infra/public/pages/metrics/layouts/nginx.ts

```diff
@@ -20,7 +21,9 @@ export const nginxLayoutCreator: InfraMetricLayoutCreator = theme => [
     sections: [
       {
         id: InfraMetric.nginxHits,
-        label: 'Hits',
+        label: i18n.translate('xpack.infra.nginxMetrics.hitsSectionLabel', {
```

Comment:
Applying the scheme would make this `xpack.infra.metricDetailPage.nginxMetricsLayout.layoutLabel` with predictable consequences for the ids below.